src/app/shared/services/user-session.service.ts

```diff
@@ -25,6 +25,13 @@ export class UserSessionService implements OnDestroy {
     distinctUntilChanged((u1, u2) => u1.groupId === u2.groupId)
   );
 
+  group$ = this.session$.pipe(
```

Comment:
1. You need to rebase from master to have #426 which did a lot of changes to this component.
2. You cannot call it `group` and it is not the group of the user, it is the watchedGroup. 
2. If you filter the `isNotNullOrUndefined` from watched group, you will have a value with just correspond to the latest watched group, which seems to be a strange value to follow.

Reply:
Fixed